libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -10,7 +10,9 @@ export enum ReadingListActions {
   ConfirmedAddToReadingList = '[Reading List API] Confirmed add to list',
   RemoveFromReadingList = '[Books Search Results] Remove from list',
   FailedRemoveFromReadingList = '[Reading List API] Failed remove from list',
-  confirmedRemoveFromReadingList = '[Reading List API] Confirmed remove from list'
+  confirmedRemoveFromReadingList = '[Reading List API] Confirmed remove from list',
```

Comment:
Better to not use action type constants, just inline them in the action creators.